tests/python/quantization/test_quantization.py

```diff
@@ -244,7 +245,8 @@ def check_quantized_pooling(data_shape, kernel, pool_type, pad, stride, global_p
         quantized_pooling = mx.sym.contrib.quantized_pooling(data=qdata, min_data=min_data,
                                                                 max_data=max_data, kernel=kernel,
```

Comment:
May be also good to fix the alignment from L246-L249.

Reply:
Fixed.